### PR TITLE
Add Google Keep menubar app v0.4.0

### PR DESCRIPTION
**Pull request by FrancesCoronel**
This is a macOS app that allows you to access [Google Keep](https://www.google.com/keep/) in your menubar.

It was created by [Travers McInerney](https://github.com/tmcinerney/keep) but I like it so much, I wanted to add it to my dotfiles but then I realized there wasn't a cask so here I am.

---

Signed-off-by: Frances Coronel <hello@fvcproductions.com>

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by fvcproductions** [comment]
Thanks for adding those changes @commitay!

File: Casks/keep.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  version '0.4.0'
+  sha256 'e7a750deb250f21ef8b0e11bb01c62299d1574779bcfc83d74c5e238f1e40889'
+
+  url "https://github.com/tmcinerney/keep/releases/download/v#{version}/keep.v#{version}.zip"
+  appcast 'https://github.com/tmcinerney/keep/releases.atom',
+          checkpoint: 'dfbb63243f9d2c22b82e0b9fca0b5c0c0956be55feb19e77b341d0de3662f0a7'
+  name 'Keep'
+  homepage 'https://github.com/tmcinerney/keep/'
+
+  auto_updates true
```
Comment (commitay) [inline Casks/keep.rb]: Are you sure that this self updates?

Comment (fvcproductions) [reply Casks/keep.rb]: No, I wasn't 100% sure. 

I might have misunderstood what auto_updates refers to.

I thought it meant the `atom` feed was automatically updated.

Comment (commitay) [reply Casks/keep.rb]: No, `auto_updates` excludes the Cask from `brew cask outdated`. It should only be added to Cask that have a built in update mechanism.

Comment (fvcproductions) [reply Casks/keep.rb]: Thanks for the clarification - noted.

**Pull request merged**